src/System.Net.HttpListener/tests/HttpResponseStreamTests.cs

```diff
@@ -478,8 +478,7 @@ public void Dispose()
 
                 // Disconnect the Socket from the HttpListener.
                 client.Close();
-                GC.Collect();
-                GC.WaitForPendingFinalizers();
+                Thread.Sleep(2000);
```

Comment:
Yup, reverted this commit.
I'm willing to delete these if there's *really* no way to check this. But it does test an important piece of functionality of HttpListener - the `IgnoreWriteExceptions` feature